### ISSUE TITLE
Feature add: a corner that prevents power settings from turning off screen

### ISSUE THREAD

**Issue by Massbadetti** [AsishRaju/HotCorners#2]
A feature I'm looking for is a corner that prevents the screen from shutting down either by screensaver or from power settings.

**Comment by AsishRaju** [comment]
@Massbadetti you mean instead of _screen off_ you want a option to enable screen saver?

**Comment by Massbadetti** [comment]
does screen off do that? My bad, I understood "Screen Off" as "It turns the Screen Off".

**Comment by AsishRaju** [comment]
@Massbadetti **_screen off_**  turns off your monitor it doesn't  enables screen saver, so do you want a option to enable screen saver rather than turning off your monitor?

**Comment by Massbadetti** [comment]
No, the opposite. What I'd like to have is a hot corner action where the screen does **not** turn off. For example, I have specified in power settings that the monitor turns off after 3 minutes of inactivity. But sometimes I might leave my PC rendering or copying something and I need the monitor turned on even if I'm not using it. That's where an action like that would be helpful: I start the rendering, park the mouse in the hot corner and the screen stays on until I take the mouse out of that corner. I am now using a third party app (Caffeinated) just for this but a hot corner action would be even better.

**Comment by AsishRaju** [comment]
@Massbadetti thanks for clarifying buddy, will definitely add this option but cant promise a deadline for it,all i can promise for is i will add this soon
so this i what i understood: a option to keep your screen on even when your screen turns off due to inactivity

**Comment by Massbadetti** [comment]
exactly! No rush, take your time, I just proposed a feature and thanks for the replies!
